### ISSUE TITLE
Implement the cross reference like the TSK

### ISSUE THREAD

**Issue by agrogers** [AndBible/and-bible#1021]
I like the new cross reference popup. Actually I have never really look at them because it was a bit too inconvenient. This will make it better for me. 

But my preference would be to display it like you do the TSK. That is, display it in its own commentary like window. That would make accessing those cross references really easy.

Video here:
https://drive.google.com/file/d/15iXDM8pzmcXUTWvmeJq71PHG0kxFjOSt/view?usp=sharing

**Comment by tuomas2** [comment]
Good idea but implementationally bit challenging as crossreferences are part of OSIS xml document, which will be parsed in js side (frontend).

**Comment by agrogers** [comment]
Ah, ok. I guess converting them to a standard commentary document would be
the way to go and possibly require less work. I should look at that one day
:)

___________________________
*www.TenOutOfTen.org* <https://www.TenOutOfTen.org>
***@***.***
(+61) 0477 174 193 (Australia)
(+95) 09 250018669 (Myanmar)

**Comment by tuomas2** [comment]
Hmm, maybe I need to take my words back. It would be easy(ish) to open the same document fragment and use a different rendering mode js side, which would display only x-refs and nothing else. Not sure what kind of UX would be good for it though. I guess codewise straight forward implementation would be to make it text workspace&window specific text display settings option flag that enables such a mode, but that might not be best UX for such a special mode.
